.github/workflows/package-publish.yml

```
@@ -0,0 +1,105 @@
+        run: |
+          tag="${GITHUB_REF#refs/tags/}"
+          tag="${tag#v}"
+          version="$(jq -r .version "${{ inputs.package-dir }}/manifest.json")"
+          test "$tag" = "$version" || {
+            echo "git tag $tag 与 manifest.version $version 不一致" >&2
+            exit 1
+          }
+      - name: 打包并冒烟安装验证
+        id: pack
+        shell: bash
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:9:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)